ApexOSA/polarimeter.py

```
@@ -0,0 +1,226 @@
+
+            for v in Str.split(" "):
+                try:
+                    Values.append(float(v))
+                except:
+                    pass
+        else:
+            S = []
+            S.append(1.0)
+            for i in range(3):
+                S.append(2.0 * random() - 1.0)
+            S0 = sqrt(S[0] ** 2 + S[1] ** 2 + S[2] ** 2)
```

Ruff (0.12.2)

Local variable `S0` is assigned to but never used

Remove assignment to unused variable `S0`

(F841)